src/config.ts

```diff
@@ -69,7 +69,7 @@ const getIssuerEndpoint = async (suffix?: string) => {
 
 const getPatientPickerEndpoint = async (suffix?: string) => {
```

Comment:
I am a little confused as to why this function is called below to set the auth and token URLs. Why aren't they calling `getIssuerEndpoint` instead? Where is the patientPickerEndpoint used?

Reply:
I will let them call getIssuerEndpoint  instead